README.md

```diff
@@ -100,10 +100,139 @@ This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0
         python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]
 
 
-    More information about the ISIS3DATA environment variable and the ISIS3 Data Area can be found [here]("#The-ISIS3-Data-Area"). Now everytime the isis3 environment is activated, $ISISROOT, $ISIS3DATA, and $ISIS3TESTDATA will be set to the values passed to isis3VarInit.py. This does not happen retroactively, re-activate the isis3 envionment with one of the following commands:
+    More information about the ISIS3DATA environment variable and the ISIS Data Area can be found [here]("#The-ISIS3-Data-Area"). Now everytime the isis environment is activated, $ISISROOT, $ISIS3DATA, and $ISIS3TESTDATA will be set to the values passed to isis3VarInit.py. This does not happen retroactively, re-activate the isis envionment with one of the following commands:
```

Comment:
`#The-ISIS3-Data-Area` is a dead link now. The proper link is `#The-ISIS-Data-Area`

Reply:
in danger of correcting too many things at once, what about the ISIS3DATA env variable name? Guess it's used SOO widely, no chance to change that anytime soon without breaking everything, right?

Reply:
@jessemapel, link fixed in upcoming commit.

@michaelaye, dude, what to name environment variables in the software suite is out-of-scope for a PR on changes to the README!